chapter09/mssw/patterns.dat

```diff
@@ -0,0 +1,10 @@
+blarg
+honk
+cocoa?n[ue]t
+Mary[−\s]+Anne?
+(The\s+)?(Skipper|Professor)
+triggered
+乒乓
+^$
+I think the next regex breaks Regexp::Assemble\.\.\.
+(?:\b(?:4[0-9]{12}(?:[0-9]{3})?|5[1-5][0-9]{14}|6(?:011|5[0-9][0-9])[0-9]{12}|3[47][0-9]{13}|3(?:0[0-5]|[68][0-9])[0-9]{11}|(?:2131|1800|35\d{3})\d{11}))
```

Comment:
This looks like it's already been through a regex assembler, no wonder you broke it.